Pages/Index.cshtml

```diff
@@ -1,126 +1,125 @@
 @page
 @model IndexModel
 @{
-    ViewData["Title"] = "Home page";
+    ViewData["Title"] = "Property Investment Home";
 
     var properties = (PropertyQuickType.Property[])ViewData["Property"];
     //var healthcares = (QuickType1.Healthcare[])ViewData["Healthcare"];
 }
 
-<main role="main">
-    
 
-    <div class="bd-example">
-        <div id="carouselExampleCaptions" class="carousel slide" data-ride="carousel">
-            <ol class="carousel-indicators">
-                <li data-target="#carouselExampleCaptions" data-slide-to="0" class="active"></li>
-                <li data-target="#carouselExampleCaptions" data-slide-to="1"></li>
-                <li data-target="#carouselExampleCaptions" data-slide-to="2"></li>
-                <li data-target="#carouselExampleCaptions" data-slide-to="3"></li>
-            </ol>
-            <div class="carousel-inner">
-                <div class="carousel-item active">
-                    <img src="/images/neighborhood1.jpg" class="d-block w-100" alt="Home" width="500" height="300">
-                    <div class="carousel-caption d-none d-md-block">
-                        <h5 class="text-center">Residential Property</h5>
-                    </div>
+
+<div class="bd-example">
```

Comment:
Adjusted indentations

Reply:
For reformatting... 
1) A caution.  Significant reformats can make future commit diffs difficult, as git has to deal with significant reordering of the logic.
2) The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.  This ameliorates issue 1, above.